README.md

```diff
@@ -1,50 +1,56 @@
-- please open an issue that requests to be added as a contributor or ping melsumner on Ember Discord
-- make sure you have 2FA set up for GitHub
+
+- Please open an issue that requests to be added as a contributor or ping `@Melanie#1618` on Ember Discord.
+- Make sure you have two-factor authentication (2FA) set up and enabled for GitHub.
 
 Once we've had a chat, we can add you as a contributor. Easy-peasy!
 
 ## Standardizing Addon Maintenance
 
 By standardizing some aspects of maintaining addons we can ensure a consistent experience for contributors and maintainers.
 
-* If your addon does not have a changelog, consider adding one
-	- Changelogs are important to communicate to end users what is going on with the project. See [keepachangelog.com](https://keepachangelog.com/en/1.0.0/) for additional reasoning.
-	- Robert Jackson (@rwjblue) has automated the initial setup of automatic changelog generation using some popular tools. Follow the steps in [create-rwjblue-release-it-setup](https://github.com/rwjblue/create-rwjblue-release-it-setup). You can see an example of how that works in [ember-pikaday](https://github.com/adopted-ember-addons/ember-pikaday).
+- If your addon does not have a changelog, consider adding one.
+  - Changelogs are important to communicate to end users what is going on with the project. See [keepachangelog.com](https://keepachangelog.com/en/1.0.0/) for additional reasoning.
+  - Robert Jackson (`@rwjblue`) has automated the initial setup of automatic changelog generation using some popular tools. Follow the steps in [create-rwjblue-release-it-setup](https://github.com/rwjblue/create-rwjblue-release-it-setup). You can see an example of how that works in [ember-pikaday](https://github.com/adopted-ember-addons/ember-pikaday).
+
+- Use standard labels for issues and pull requests.
+  - Defer to the learning team's recommendations and [lerna-changelog](https://github.com/lerna/lerna-changelog)'s defaults.
 
-* Use standard labels for issues and pull requests.
-	* Defer to the learning team's recommendations and [lerna-changelog](https://github.com/lerna/lerna-changelog)'s defaults.
+- Add a code of conduct to the repository.
+  - You can read how to add the Contributor Covenant here: [https://www.contributor-covenant.org/](https://www.contributor-covenant.org/)
 
-* Add a code of conduct to the repo
-	- You can read how to add the Contributor Covenant here: [https://www.contributor-covenant.org/](https://www.contributor-covenant.org/)
-	
 ## Hosting and Deployments
 
 While we don't enforce a single way to host and deploy addon docs, many of the addons in this org use Netlify to deploy and host the addon docs.
 
-<a href="https://www.netlify.com">
-  <img src="https://www.netlify.com/img/global/badges/netlify-color-accent.svg" alt="Deploys by Netlify" />
-</a>
+[
+  ![Deploys by Netlify](https://www.netlify.com/img/global/badges/netlify-color-accent.svg)
+](https://www.netlify.com)
```

Comment:
Unfortunate that this removed the `alt`

Reply:
Let's not do it if it removes the `alt`.

Reply:
I have double-checked that it still renders the `alt` attribute with `"Deploys by Netlify"`. The rendered HTML markup is:

```html
<p><a href="https://www.netlify.com" rel="nofollow">
<img src="https://camo.githubusercontent.com/89bdfa968b425fbf4502e6a0ccd2567501dd82bf/68747470733a2f2f7777772e6e65746c6966792e636f6d2f696d672f676c6f62616c2f6261646765732f6e65746c6966792d636f6c6f722d616363656e742e737667" alt="Deploys by Netlify" data-canonical-src="https://www.netlify.com/img/global/badges/netlify-color-accent.svg" style="max-width:100%;">
</a></p>
```

The `alt` attribute is set by the content in square brackets. The image syntax in markdown is: `![alt text](image url)`.